### PR TITLE
vsce: add release steps and update feedback link

### PR DESCRIPTION
**Pull request by abeatrix**
Closes https://github.com/sourcegraph/sourcegraph/issues/34742 https://github.com/sourcegraph/sourcegraph/issues/34822

This PR adds release steps for the extension and update the links for submitting feedback and feature request

## Test plan

Docs and links update

## App preview:

- [Web](https://sg-web-bee-vsce-release-doc.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-xmoxhtmbkl.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -55,3 +55,19 @@ All open issues are not yet solved. If the task is interesting to you, take it a
 - Git branch name convention: `[developer-initials]/short-feature-description`
 - [Examples on Github](https://github.com/sourcegraph/sourcegraph/pulls?q=is%3Apr+label%3Ateam%2Ffrontend-platform)
 - (For Sourcegraph team) [How to accept contributions](https://docs.sourcegraph.com/dev/contributing/accepting_contribution)
+
+### Release Process
```
Comment (vdavid) [inline CONTRIBUTING.md:59]: Are you sure you wanted to add this to the top-level CONTRIBUTING.md? If yes then I think it'd be better to add these under a "VS Code" heading, or maybe just link to client/vscode/CONTRIBUTING.md under a VS Code heading.

File: client/vscode/CONTRIBUTING.md [hunk 2]
```diff
@@ -46,16 +46,27 @@ If you need guidance or have any questions regarding Sourcegraph or the extensio
...
 
 Apache
 
-## Release
+### Release Process
+
+The release process for the VS Code Extension for Sourcegraph is currently automated.
 
-The extension is released to the VS Code Marketplace automatically whenever the `vsce/release` branch is updated.
+#### Release Steps:
```
Comment (vdavid) [inline client/vscode/CONTRIBUTING.md:62]: Nit: remove the colon because this is a header, like a title?

**Review by vdavid — COMMENTED** [review]
I don't have a lot of context so I don't feel qualified to approve this, so just left a few comments, I hope it helps! :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (abeatrix) [reply CONTRIBUTING.md:59]: Nice catch! I made the changes to the wrong file initially and thought I've removed them so thanks for catching this!

**Review by vdavid — APPROVED** [review]
LGTM :) I love docs updates <3

**Review by ryankscott — APPROVED** [review]
LGTM - sorry about the delay...

**Pull request merged**